### PR TITLE
Add direct link to the Github issue repo from error pages

### PR DESCRIPTION
**Pull request by dennisschagt**
Fixes [issue #165](https://github.com/mozilla-iam/dino-park-issues/issues/165).

### CODE REVIEW

File: public/error.html [hunk 1]
```diff
@@ -41,8 +41,11 @@ <h2>This page isn't available</h2>
...
             <a href="/" class="button router-link-active">Go to homepage</a>
             <p>
-              <small
-                >Please submit all bugs or issues to the project's GitHub issue repository (link in footer).</small>
+              <small>
+                Please submit all bugs or issues to
+                <a href="https://github.com/mozilla-iam/dino-park-issues">the
+                project's GitHub issue repository</a>.
+              </small>
             </p>
```
Comment (LeoMcA) [inline public/error.html]: From what I can see, this whole file isn't used anywhere - if you open people.mozilla.org/error.html directly, you'll see it's very broken - so I think we can just delete the whole file.

File: src/pages/PageSearchResult.vue [hunk 2]
```diff
@@ -21,10 +21,11 @@
         <h2>This page isn't available</h2>
         <p>An error occured while trying to load the search results</p>
         <p>
-          <small
-            >Please submit all bugs or issues to the project's GitHub issue
-            repository (link in footer).</small
-          >
+          <small>
+            Please submit all bugs or issues to
+            <a href="https://github.com/mozilla-iam/dino-park-issues">the
```
Comment (LeoMcA) [inline src/pages/PageSearchResult.vue]: Can we use the value from `LinksMixin` instead? You can see how to include a link in this commit: https://github.com/mozilla-iam/dino-park-front-end/commit/9dacc91357ae3b337337ac421742a99488d72cf1#

File: src/pages/PageUnknown.vue [hunk 3]
```diff
@@ -21,10 +21,11 @@
           >Go to homepage</RouterLink
         >
         <p>
-          <small
-            >Please submit all bugs or issues to the project's GitHub issue
-            repository (link in footer).</small
-          >
+          <small>
+            Please submit all bugs or issues to
+            <a href="https://github.com/mozilla-iam/dino-park-issues">the
```
Comment (LeoMcA) [inline src/pages/PageUnknown.vue]: Same as above

File: src/pages/PageSearchResult.vue (hunk 2 above)
Comment (dennisschagt) [reply src/pages/PageSearchResult.vue]: Suggestion applied in both instances.

Thanks for the review!

**Comment by LeoMcA** [comment]
Oops, looks like you need to sign your commits before we can merge. Github has instructions on how to do this here: https://help.github.com/en/github/authenticating-to-github/about-commit-signature-verification#gpg-commit-signature-verification

You might find keybase provides a nicer interface for generating pgp keys: https://keybase.io/docs/command_line

**Comment by dennisschagt** [comment]
Alright, thanks for the instructions.

The commit is signed now.

**Comment by LeoMcA** [comment]
Thanks! I've deployed it to prod, you can see it in action here: https://people.mozilla.org/404

**Pull request merged**